src/SDKs/_metadata/resources_resource-manager.txt

```diff
@@ -1,41 +1,11 @@
-﻿Installing AutoRest version: latest
-
-> autorest@2.0.4280 preinstall C:\Users\tiano\AppData\Roaming\npm\node_modules\autorest
-> node ./preinstall-check
-
-C:\Users\tiano\AppData\Roaming\npm\autorest -> C:\Users\tiano\AppData\Roaming\npm\node_modules\autorest\dist\app.js
-+ autorest@2.0.4280
-updated 1 package in 2.1s
-
-AutoRest installed successfully.
-Commencing code generation
-Generating CSharp code
-Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\code\azure-sdk-for-net\src\SDKs
-AutoRest code generation utility [version: 2.0.4280; node: v8.11.2]
-(C) 2018 Microsoft Corporation.
-https://aka.ms/autorest
-   Loading AutoRest core      'C:\Users\tiano\.autorest\@microsoft.azure_autorest-core@2.0.4280\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4280)
-   Loading AutoRest extension '@microsoft.azure/autorest.csharp' (~2.2.51->2.2.57)
-   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.43->2.3.43)
-Processing batch task - {"package-features":true} .
```

Comment:
I believe the SDK was earlier generated using batch tasks (multi-api flag?) did something change since? 
Please go through the files and check if any files failed to generated or were accidentally deleted

Reply:
@dsgouda This was generated using the same mechanism (batch + generate.cmd).  However, generate.cmd failed during the metadata generation so I manually updated the metadata txt file.  I verified that the generation is correct (by running autorest manually on master on the full batch).

This generated code does contain minor *Client.cs changes (due to a newer autorest version) to all the clients that are generated via the batch run in /Resources.  If it wasn't batch we'd only see changes to the PolicyClient (and policy related models).

Reply:
The txt file must not be modified manually. If the code was generated using some additional flags in generate.cmd you can simply add those to the `generate.ps1` file as `AutoRestCodeGenerationFlags` 
eg: 
```
Start-AutoRestCodeGeneration -AutoRestCodeGenerationFlags "--multi-api --some-other-flag"
```
Please regenerate the code using `generate,ps1` file again and update the PR

Reply:
@dsgouda It looks like the metadata generation PS code changed since the last time this project was generated.  An exception is hit on line 302 of AutoRestCodeGenerationModule.psm because the $sdkInfo variable is an array of contents from multiple files instead of a single file.  This new implementation is not going to work for any project that has multiple SdkInfo_*.cs files (as we do)...  

Write-ErrorLog : System.Management.Automation.MethodInvocationException: Exception calling "Insert" with "2" argument(s): "Collection was of a fixed size." ---> System.NotSupportedException: Collection was
of a fixed size.
   at System.Array.System.Collections.IList.Insert(Int32 index, Object value)
   at CallSite.Target(Closure , CallSite , Object , Object , Object )
   --- End of inner exception stack trace ---
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.Interpreter.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.LightLambda.RunVoid1[T0](T0 arg0)
   at System.Management.Automation.PSScriptCmdlet.RunClause(Action`1 clause, Object dollarUnderbar, Object inputToProcess)
   at System.Management.Automation.PSScriptCmdlet.DoEndProcessing()
   at System.Management.Automation.CommandProcessorBase.Complete()

Debugging this wasn't a fun experience.  How do we move forward here?

Reply:
@pilor apologize for that, I will take a look at the `generate.ps1`

Reply:
@pilor Please pull down latest changes from psSdkJson6, run `git clean -xdff` and run `msbuild build.proj`
Regenerating the code should succeed once this is done

Reply:
@dsgouda Done.  No more errors but the metadata text file ends up empty.  No generated code changes (the resulting autorest command is the same as what I was running before).